src/offline.js

```diff
@@ -1,8 +1,9 @@
 import PouchDB from 'pouchdb'
 import pouchdbFind from 'pouchdb-find'
-PouchDB.plugin(pouchdbFind)
 import { DOCTYPE_FILES } from './doctypes'
 
+let pluginLoaded = false
```

Comment:
Why is `pluginLoaded` a "global" variable and not a `createDatabase()` variable?